src/pydicom/data/data_manager.py

```diff
@@ -326,7 +326,18 @@ def get_testdata_file(
     str, pydicom.dataset.Dataset or None
         The absolute path of the file if found, the dataset itself if `read` is
         ``True``, or ``None`` if the file is not found.
+
+    Raises
+    ______
+    ValueError
+        If `name` is an absolute path.
     """
+    if os.path.isabs(name):
```

Comment:
`Path(name).is_absolute()` didn't work for some reason, I didn't investigate why

Reply:
For fun, I went to see how [pathlib does it](https://github.com/python/cpython/blob/3f84a19e6291db682fc9a570e7612e80e2ffbbb5/Lib/pathlib.py#L1107) - it raises on `pattern.drive or pattern.root`.  So we could do that and definitely avoid the error.  But I'm fine with this too.

Reply:
Yes, this should have the same outcome.